web/src/lib/stores/unique-id.store.ts

```diff
@@ -0,0 +1,16 @@
+import { get, writable } from 'svelte/store';
+
+function createIdStore() {
+  const { subscribe, update } = writable(-1);
+
+  return {
+    subscribe,
+    update,
+    generateId: () => {
+      update((value) => value + 1);
+      return `id-${get(uniqueIdStore)}`;
+    },
+  };
+}
+
+export const uniqueIdStore = createIdStore();
```

Comment:
This doesn't need to be a store and it doesn't need to be subscribed to.

It could be a pain exported function with local state. Or, a generator.

Reply:
Good call, that would be much simpler. Working on a fix.